shell/components/form/NameNsDescription.vue

```diff
@@ -215,11 +217,15 @@ export default {
 
   computed: {
     ...mapGetters(['currentProduct', 'currentCluster', 'namespaces', 'allowedNamespaces']),
+    ...mapGetters('customisation', ['getPrewCluster']),
     namespaceReallyDisabled() {
       return (
         !!this.forceNamespace || this.namespaceDisabled || this.mode === _EDIT
       ); // namespace is never editable
     },
+    clusterPreview() {
+      return this.getPrewCluster;
```

Comment:
This surely is "easy" to get but I am not able to get it at this moment... What does `prew` mean? 😅